src/library/dao/trackdao.cpp

```diff
@@ -1293,6 +1303,38 @@ TrackPointer TrackDAO::getTrackFromDB(TrackId trackId) const {
         }
     }
 
+    // Data migration: Reload track total from file tags if not initialized
+    // yet. The added column "tracktotal" has been initialized with the
+    // default value "//".
+    // See also: Schema revision 26 in schema.xml
+    if (pTrack->getTrackTotal() == "//") {
+        // Just in case: Use track total from the track number itself if
+        // it can be split into two parts and the second part os not empty.
+        const std::pair<QString, QString> splittedTrackNumber(
+                TrackNumbers::splitString(pTrack->getTrackNumber()));
+        if (splittedTrackNumber.second.isEmpty()) {
+            // Reload track total from file tags
+            Mixxx::TrackMetadata trackMetadata;
+            if (OK == readTrackMetadataAndCoverArtFromFile(&trackMetadata, nullptr, location)) {
+                pTrack->setTrackTotal(trackMetadata.getTrackTotal());
+                // Also set the track number if it is still empty due
+                // to insufficient parsing capabilities of Mixxx in
+                // previous versions.
+                if (!trackMetadata.getTrackNumber().isEmpty() && pTrack->getTrackNumber().isEmpty()) {
+                    pTrack->setTrackNumber(trackMetadata.getTrackNumber());
+                }
+            } else {
+                qWarning() << "Failed to reload track number/total from file tags:"
+                        << location;
+            }
+        } else {
+            // Initialize track total with the value stored in the Mixxx library
```

Comment:
I can't get the point why we can reach this. 
We have the TrackNumber column which does not contain a "1/12" format and we have the new metadata code which also not produces a "1/12" format.